src/services/auth/setAuthAPI.ts

```diff
@@ -1,21 +1,28 @@
-import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
-import { AUTH, GOOGLE, SIGN_IN, SIGN_UP } from "utils/consts/breakepointConsts";
+import apiSlice from "services/api/apiSlice";
+import {
+  AUTH,
+  GOOGLE,
+  LOG_OUT,
+  SIGN_IN,
+  SIGN_UP,
+} from "utils/consts/breakpointConsts";
 
 interface AuthFields {
   email: string;
   password: string;
 }
 
 export interface AuthResponse {
-  token: string;
-  email: string;
-  id: number;
-  role: "freelancer" | "client" | undefined;
+  accessToken: {
+    authToken: string;
+    email: string;
+    id: number;
+    role: "freelancer" | "client" | undefined;
```

Comment:
why do u need specify `undefined`?
 `role?: "freelancer" | "clien"` should give same type, since question mark make this field optional